artview/components/plot.py

```diff
@@ -61,13 +61,14 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, \
         parent - PyQt instance
             Parent instance to associate to Display window.
             If None, then Qt owns, otherwise associated with parent PyQt instance.
-        
+
         Notes::
         -----
         This class records the selected button and passes the 
         change value back to variable.
         '''
         super(Display, self).__init__(name=name, parent=parent)
+        self.setFocusPolicy(QtCore.Qt.ClickFocus)
```

Comment:
Okay, I wondered because I never experienced that on my machine. We may run across more as users try this package out.